src/fiskaltrust.Launcher/Constants/Runtime.cs

```diff
@@ -0,0 +1,30 @@
+namespace fiskaltrust.Launcher.Constants
+{
+    public class Runtime
+    {
+        public static string Identifier
+        {
+            get
+            {
+                string runtimeIdentifier = Environment.Is64BitProcess ? "x64" : "x86";
```

Comment:
Should we also take care of Arm? 🤔

Reply:
In other places, we use `RuntimeInformation.ProcessArchitecture` to detect this

Reply:
ATM these are the platforms we build for... we'll need to update the pipelines as well. Maybe create an issue for ARM generally because it is not tested there.

Reply:
Good point, created issue #48